crates/nu-protocol/src/span.rs

```diff
@@ -30,6 +30,8 @@ impl<T> IntoSpanned for T {
     }
 }
 
+impl<T> Copy for Spanned<T> where T: Copy {}
```

Comment:
You can just add `#[derive(Copy)]` to `Spanned`, it will automatically add the `where T: Copy` constraint.

Reply:
Thanks!  I also find it's already added on latest main